front-end/src/TermsOfService.css

```diff
@@ -1,32 +1,42 @@
 h1 {
     margin-top: 0;
     padding: 0;
-}
+    color: #D1CDC7; 
```

Comment:
Great work on the terms of service styling. In the future, this should probably be done in a separate branch than the one you were using for working on the individual show page, but I applaud the initiative.